### PR TITLE
ING-1295: Remove manifestID from OrchestrateMemdCollectionID cb

### PR DESCRIPTION
**Pull request by chvck**

### CODE REVIEW

File: crud_test.go [hunk 1]
```diff
@@ -62,21 +62,14 @@ func TestSimpleCrudCollectionMapOutdatedRetries(t *testing.T) {
...
-	)
+	res, err := OrchestrateSimpleCrud[*UpsertResult](context.Background(), rs, cr, vb, nil, nkcp, "scope", "collection", 0, []byte("somekey"), fn)
 	require.NoError(t, err)
 	require.NotNil(t, res)
 
 	assert.Equal(t, 2, collectionCalls)
 	assert.Equal(t, 2, fnCalls)
 }
+
+func TestSimpleCrudCollectionIDOnlyReturnsImmediately(t *testing.T) {
```
Comment (brett19) [inline crud_test.go:73]: :O

**Review by brett19 — CHANGES_REQUESTED** [review]

File: vbucketconsistency.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+				RequestedVbId: vbID,
+				RequestVbUUID: vbUUID,
+				ActualVbUUID:  serverVbUUID,
+			}
+		}
+
+		return nil
+	}
+
+	vb.slowLock.Lock()
```
[the remark this answers is not in the document]
Comment (chvck) [reply vbucketconsistency.go:51]: It prevents us from writing  2 different caches into the baggage under the same key, which could lead to racey behaviour where 2 calls to Baggage(...) return different caches. Possibly unimportant but seems edge casey and hard to debug.

File: crud_test.go (hunk 1 above)
Comment (chvck) [reply crud_test.go:73]: Are you questioning the quality of my excellent test? It's guaranteed to pass!

Comment (brett19) [reply crud_test.go:73]: My mistake. +2
